osmaxx-py/osmaxx/excerptexport/forms/export_options_form.py

```diff
@@ -1,16 +1,18 @@
 from django import forms
-
-from osmaxx.excerptexport import settings
+from django.utils.translation import ugettext_lazy as _
 
 
 class ExportOptionsForm(forms.Form):
     max_items_radio = 5
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, converter_configuration, *args, **kwargs):
         super(ExportOptionsForm, self).__init__(*args, **kwargs)
-        for export_option_key, export_option in settings.EXPORT_OPTIONS.items():
+        self.converter_configuration = converter_configuration
+
+        for export_option_key, export_option in self.converter_configuration.items():
             self.create_checkboxes(
-                'export_options.'+export_option_key+'.formats', export_option['name'],
+                'export_options.'+export_option_key+'.formats',
```

Comment:
Put spaces around operator `+`. (Doesn't `flake8` complain about the missing spaces?)